Assets/SEE/UI/DebugAdapterProtocol/DebugAdapterProtocolManager.cs

```diff
@@ -0,0 +1,263 @@
+            fileNameProperty.Name = "File Name";
+            fileNameProperty.Description = "The file name (executable) of the debug adapter.";
+            fileNameProperty.FallbackDirectory = adapter.AdapterWorkingDirectory;
+            group.AddProperty(fileNameProperty);
+
+            StringProperty argumentsProperty = go.AddComponent<StringProperty>();
+            argumentsProperty.Name = "Arguments";
+            argumentsProperty.Description = "The arguments of the debug adapter.";
+            group.AddProperty(argumentsProperty);
+
+            SelectionProperty cityProperty = go.AddComponent<SelectionProperty>();
+            cityProperty.AddOptions(cities.Select(c => c.name));
+            cityProperty.Value = city.name;
+            cityProperty.Description = "City used where current execution position is highlighted.";
+            group.AddProperty(cityProperty);
+
+            OnAdapterChanged();
+            debugAdapterProperty.OnComponentInitialized += () =>
+            {
+                debugAdapterProperty.horizontalSelector.selectorEvent.AddListener(_ =>
+                {
+                    UpdateValues();
+                    adapter = adapters[debugAdapterProperty.horizontalSelector.index];
+                    OnAdapterChanged();
+                });
+            };
+
+            // create dialog
+            PropertyDialog.PropertyDialog dialog = go.AddComponent<PropertyDialog.PropertyDialog>();
+            dialog.Title = "Debug Adapter";
+            dialog.Description = "Enter the debug adapter configuration.";
+            dialog.AddGroup(group);
+            dialog.OnConfirm.AddListener(UpdateValues);
+
+            // open dialog
+            SEEInput.KeyboardShortcutsEnabled = false;
+            dialog.OnCancel.AddListener(OnClose);
+            dialog.OnConfirm.AddListener(OnClose);
+            dialog.DialogShouldBeShown = true;
+
+            void OnClose()
+            {
+                SEEInput.KeyboardShortcutsEnabled = true;
+                Destroyer.Destroy(go);
+            }
+            void OnAdapterChanged()
+            {
+                workingDirectoryProperty.Value = adapter.AdapterWorkingDirectory;
+                fileNameProperty.Value = adapter.AdapterFileName;
+                argumentsProperty.Value = adapter.AdapterArguments;
+            }
+            void UpdateValues()
+            {
+                adapter.AdapterWorkingDirectory = workingDirectoryProperty.Value ;
+                adapter.AdapterFileName = fileNameProperty.Value ;
+                adapter.AdapterArguments  = argumentsProperty.Value ;
+                city = cities.First(c => c.name == cityProperty.Value);
+            }
+        }
+
+        /// <summary>
+        /// Opens the launch configuration dialog.
+        /// </summary>
+        public static void OpenLaunchConfig()
+        {
+            GameObject go = new GameObject("Launch Request");
```

Comment:
That is (unfortunately) not an option:
The current game object is the container for the code window (or other windows) and gets disabled/destroyed when the window closes/changes/etc.
Since the debug session should proceed that, another game object is required.